plugins/com.google.cloud.tools.eclipse.appengine.compat/xslt/wtpMetadata.xsl

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  This stylesheet removes the following GPE runtime and facets from the WTP facet metadata file:
+
+    <runtime name="Google App Engine"/>
+    <installed facet="com.google.appengine.facet" version="1"/>
+    <installed facet="com.google.appengine.facet.ear" version="1"/>
+
+  The metadata file is ".settings/org.eclipse.wst.common.project.facet.core.xml".
+-->
+<xsl:stylesheet version="2.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
```

Comment:
I don't know what the processor means here, but from what you said, I guess XSLT 1 is sufficient. Changed the version to "1.0".